doc/Command-Reference.md

```diff
@@ -10713,6 +10713,35 @@ This command is used to disable syslog rate limit feature.
   config syslog rate-limit-feature disable database -n asci0
   ```
 
+**config syslog level**
+
+This command is used to configure log level for a given log identifier.
+
+- Usage:
+  ```
+  config syslog level -i <log_identifier> -l <log_level> --service [<service_name>] --program [<program_name>]
+
+  config syslog level -i <log_identifier> -l <log_level> --service [<service_name>] --pid [<process_id>]
+
+  config syslog level -i <log_identifier> -l <log_level> ---pid [<process_id>]
```

Comment:
@Junchao-Mellanox better to specify "service_name" -> "container_name"  along with --container?